go/cmd/dolt/commands/diff.go

```diff
@@ -155,7 +157,13 @@ func (cmd DiffCmd) Exec(ctx context.Context, commandStr string, args []string, d
 		return HandleVErrAndExitCode(errhand.VerboseErrorFromError(err), usage)
 	}
 
-	verr := diffUserTables(ctx, fromRoot, toRoot, dArgs)
+	var verr errhand.VerboseError
```

Comment:
No reason to move this to a var declaration, just keep it in the block. These are two different variables.